src/dlls/mscoree/i386/handlers.asm

```diff
@@ -1,38 +0,0 @@
-;
-; Copyright (c) Microsoft. All rights reserved.
-; Licensed under the MIT license. See LICENSE file in the project root for full license information. 
-;
-
-; ==++==
-; 
-
-; 
-; ==--==
-;
-;  This file contains definitions for each CLR exception handler.  
```

Comment:
Moved and added comments. I noted the linker issue as well in case anyone ever tries to move these again.